starters/express-typescript-typeorm-postgres/src/middlewares/cors/index.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable prettier/prettier */
```

Comment:
do we want to disable prettier, or is this just something that is left here accidentally?